mix.exs

```diff
@@ -16,6 +17,10 @@ defmodule OAuther.Mixfile do
   defp description,
     do: "Library to authenticate with OAuth 1.0 protocol."
 
+  defp deps do
+    [{:dialyxir, "~> 0.3", only: [:dev]}]
```

Comment:
I didn't work much with any of them, but I value the code quality of "dialyze" package and believe in its author's experience.